next.config.js

```diff
@@ -3,4 +3,11 @@ const withBundleAnalyzer = require("@next/bundle-analyzer")({
   enabled: process.env.ANALYZE === "true",
 });
 
-module.exports = withBundleAnalyzer(withTM({}));
+module.exports = withBundleAnalyzer(
+  withTM({
+    reactStrictMode: true,
```

Comment:
@damassi an interesting flag to automatically apply react strict mode to the whole app.